core/iwasm/interpreter/wasm_loader.c

```diff
@@ -2716,9 +2759,15 @@ load_tag_section(const uint8 *buf, const uint8 *buf_end, const uint8 *buf_code,
                 goto fail;
             }
 
+            if (!(tag = module->tags[tag_index] = loader_malloc(
```

Comment:
Seems that the resources allocated by exception handling are not freed in wasm_loader_unload, like `module->tags[i]`, `module->tags`, could you help check?

Reply:
Agreed. Ensured tag memory clean up.